spec/graphql/mutations/wallets/update_spec.rb

```diff
@@ -34,6 +41,15 @@
           id: wallet.id,
           name: 'New name',
           expirationAt: expiration_at.iso8601,
+          recurringTransactionRules: [
+            {
+              id: recurring_transaction_rule.id,
+              ruleType: 'interval',
+              interval: 'weekly',
+              paidCredits: '22.2',
+              grantedCredits: '22.2',
+            }
```

Comment:
Style/TrailingCommaInArrayLiteral: Put a comma after the last item of a multiline array. (https://rubystyle.guide#no-trailing-array-commas)